### PR TITLE
[Docs] how to set the interval parameter

### PR DESCRIPTION
**Pull request by zhouzaida**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/zh_cn/common_usage/set_interval.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+# 设置间隔
```
Comment (C1rN09) [inline docs/zh_cn/common_usage/set_interval.md:1]:
```suggestion
# 设置日志、权重保存、验证的频率
```
我感觉这样的标题会更直观一些？不然间隔有点让人困惑

File: docs/zh_cn/common_usage/set_interval.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    train_cfg=dict(by_epoch=False, max_iters=10000, val_interval=2000),
+    val_dataloader=val_dataloader,
+    val_cfg=dict(),
+    val_evaluator=dict(type=Accuracy),
+)
+runner.train()
+```
+
+## 设置保存权重的间隔
+
```
Comment (C1rN09) [inline docs/zh_cn/common_usage/set_interval.md:48]:
```suggestion
设置 `CheckpointHook` 的 `interval` 值即可定制保存权重的间隔。
```

File: mmengine/runner/utils.py [hunk 3]
```diff
@@ -27,7 +27,7 @@ def calc_dynamic_intervals(
 
     Returns:
         Tuple[List[int], List[int]]: a list of milestone and its corresponding
-            intervals.
+        intervals.
```
Comment (C1rN09) [inline mmengine/runner/utils.py:30]: 这个空格应该是需要保留的把？

**Review by C1rN09 — APPROVED** [review]
Approve with some comments

File: mmengine/runner/utils.py (hunk 3 above)
Comment (zhouzaida) [reply mmengine/runner/utils.py:30]: Returns 是需要上下对齐的，否则渲染出来的效果不对

File: docs/zh_cn/common_usage/set_interval.md (hunk 1 above)
Comment (zhouzaida) [reply docs/zh_cn/common_usage/set_interval.md:1]: 这样比较直观，但是标题有点过长

Comment (C1rN09) [reply docs/zh_cn/common_usage/set_interval.md:1]: 设置日志/保存权重/验证的频率
或者有没有更好的？感觉“设置间隔”还是有点不太利于检索

Comment (zhouzaida) [reply docs/zh_cn/common_usage/set_interval.md:1]: “设置日志、权重保存、验证的频率”这个标题挺合适

**Pull request merged**